riscv/tests/instructions.rs

```diff
@@ -4,12 +4,17 @@ mod instruction_tests {
     use std::path::Path;
 
     use crate::common::verify_riscv_asm_file;
-    use powdr_riscv::Runtime;
+    use powdr_riscv::CompilerOptions;
     use test_log::test;
 
     fn run_instruction_test(path: &Path) {
+        let options_32 = CompilerOptions::new_32();
+        run_instruction_test_with_options(path, options_32);
```

Comment:
is the extra function needed?

Reply:
in the riscv-bb PR we also run for babybear here, then it made sense. I removed it after the extraction to keep this PR GL only